packages/fluentui/react-northstar-fela-renderer/src/felaExpandCssShorthandsPlugin.ts

```diff
@@ -22,29 +21,29 @@ const handledCssProps: Partial<Record<keyof CSS.Properties, true>> = {
 };
 
 export const felaExpandCssShorthandsPlugin = (styles: ICSSInJSStyle): ICSSInJSStyle => {
-  return Object.keys(styles).reduce((acc: ICSSInJSStyle, cssPropertyName: keyof CSS.Properties) => {
+  return Object.keys(styles).reduce((
+    // Without casting to any TSC gives "Expression produces a union type that is too complex to represent" error
+    acc: any,
+    cssPropertyName: keyof ICSSInJSStyle,
+  ) => {
     const cssPropertyValue = styles[cssPropertyName];
 
     if (cssPropertyValue === null || typeof cssPropertyValue === 'undefined') {
-      return { ...acc, [cssPropertyName]: cssPropertyValue };
-    }
-
-    if (handledCssProps[cssPropertyName]) {
+      acc[cssPropertyName] = cssPropertyValue;
+    } else if (handledCssProps[cssPropertyName]) {
       const expandedProps = expandProperty(cssPropertyName, cssPropertyValue);
 
       if (expandedProps) {
         return { ...acc, ...expandedProps };
```

Comment:
This one is kept as an alternative is `Object.assign` that is [slower](https://www.measurethat.net/Benchmarks/Show/15416/0/javascript-reduce-assign-vs-spread#latest_results_block)

Reply:
IMO it's nicer to use `Object.assign` and not rely on compilation since it's hard to say in the future if that will reman the case